app/models/item.rb

```diff
@@ -0,0 +1,4 @@
+class Item < ApplicationRecord
+    belongs_to :invoice
+    validates :invoice_id, :description, presence: true
```

Comment:
same here if you add the db belongs to then you do not need to validate presence of invoice_id.